### ISSUE TITLE
Removing items from a list

### ISSUE THREAD

**Issue by KittyGiraudel** [sass/sass#1051]
Since we are about to have a function to remove a key pair from a map (#1032), I believe that could be a good idea to keep some consistency and have the same thing for a list.

Thus, very standard list functions would be covered with `append()` to add, `set-nth()` to update and `remove()` to delete. 

And since we'll have a way to update at index `n`, I guess it could make sense to have `insert-nth` and `remove-nth` as well.

As of today, we still have to resort to custom functions or external libraries when heavily dealing with lists (thus http://sassylists.com).

Cheers,

**Comment by strarsis** [comment]
@HugoGiraudel: The link http://sassylists.com doesn't work anymore (domain seems deleted).

**Comment by KittyGiraudel** [comment]
Yup. I let it expire. Here is the link: http://at-import.github.io/SassyLists/.
